relays/client-pangolin/src/runtime.rs

```diff
@@ -29,6 +29,9 @@ pub enum Call {
 	/// Bridge rococo grandpa pallet
 	#[codec(index = 60)]
 	BridgeRococoGrandpa(BridgeRococoGrandpaCall),
+	/// Bridge pangolin parachain messages pallet
+	#[codec(index = 63)]
+	BridgePangolinParachainMessages(BridgePangolinParachainMessagesCall),
 	/// Feemarket pallet
 	#[codec(index = 53)]
 	Feemarket(FeemarketCall),
```

Comment:
The name might be `PangolinParachainFeeMarket`

Reply:
This change may alse require updates in pangolin-pangoro bridger. CC @fewensa